python/virgo/mpi/parallel_hdf5.py

```diff
@@ -98,7 +98,7 @@ def collective_read(dataset, comm, buffer_size=None):
     """
     Do a parallel collective read of a HDF5 dataset by splitting
     the dataset equally between MPI ranks along its first axis.
-    
```

Comment:
I think I've been accidentally adding trailing whitespace with the auto indenting in emacs. I should remove it all at some point but that can be another pull request.